controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -1686,10 +1686,10 @@ bool master_thread::handle_cmdu_1905_topology_notification(Socket *sd,
                              &new_event);
         }
 #endif
+
         if (database.get_node_ipv4(client_mac_str).empty()) {
             database.set_node_state(client_mac_str, beerocks::STATE_CONNECTED_IP_UNKNOWN);
             LOG(INFO) << "STATE_CONNECTED_IP_UNKNOWN for node mac " << client_mac_str;
-            return true;
```

Comment:
UNKNOWN_IP states were removed from bml in c07a34, so I maybe it could be removed from the bcl as well?

Reply:
Agreed. No need for this state anymore.